utils/items.cfg

```diff
@@ -8,6 +8,10 @@
     [/trait]
 #enddef
 
+#define WISH_ITEM_HALO
+    halo = "halo/misc/leadership-flare-1.png:20,halo/misc/leadership-flare-2.png:20,halo/misc/leadership-flare-3.png:20,halo/misc/leadership-flare-4.png:20,halo/misc/leadership-flare-4.png:20,halo/misc/leadership-flare-6.png:20,halo/misc/leadership-flare-7.png:20,halo/misc/leadership-flare-8.png:20,halo/misc/leadership-flare-9.png:20,halo/misc/leadership-flare-10.png:20,halo/misc/leadership-flare-11.png:20,halo/misc/leadership-flare-12.png:20,halo/misc/leadership-flare-13.png:20,misc/blank-hex.png:1000"
```

Comment:
the `halo=` part is a duplicate and already specified below.

Reply:
Could be shortened using the syntax in https://wiki.wesnoth.org/AnimationWML#Progressive_strings.

Reply:
Change:
```
"halo/misc/leadership-flare-[1~13]:[20*13], misc/blank-hex.png:1000"
```

Reply:
remove the `halo =` and replace the animation string with `halo/misc/leadership-flare-[1~13].png:[20*13],misc/blank-hex.png:1000"`

Reply:
Done